### PR TITLE
small change of date

### PR DESCRIPTION
**Pull request by asylves1**
# Description

* Include a summary of the changes and the related issue. 
* Include relevant motivation and context.

Resolves #(issue)

### CODE REVIEW

File: packages/client/hmi-client/src/components/home/tera-project-card.vue [hunk 1]
```diff
@@ -51,7 +51,7 @@
 				<section class="details">
 					<div>
 						<div class="author">{{ project?.userName ?? '——' }}</div>
-						<div class="creation-date">{{ formatDdMmmYyyy(project.createdOn) }}</div>
+						<div class="update-date">{{ formatDdMmmYyyy(project.updatedOn) }}</div>
```
Comment (mwdchang) [inline packages/client/hmi-client/src/components/home/tera-project-card.vue:54]: Do we even have css-classes for `creation-date` or `update-date`? If not maybe we should just remove.

Comment (YohannParis) [reply packages/client/hmi-client/src/components/home/tera-project-card.vue:54]:
```suggestion
						<div>{{ formatDdMmmYyyy(project.updatedOn) }}</div>
```

File: packages/client/hmi-client/src/page/Home.vue [hunk 2]
```diff
@@ -47,10 +47,18 @@
...
+							<div>
+								<InputText
+									v-model="searchProjects"
+									placeholder="Search for projects"
+									id="searchProject"
+									class="p-inputtext"
+								/>
+							</div>
+							<div>
 								<!-- TODO: Add project search back in once we are ready
```
Comment (YohannParis) [inline packages/client/hmi-client/src/page/Home.vue:62]: This needs to be removed if you are adding the search back.
Isn't this be done in another PR?

File: packages/client/hmi-client/src/page/Home.vue [hunk 3]
```diff
@@ -167,10 +178,14 @@ import { useProjectMenu } from '@/composables/project-menu';
...
 import { Vue3Lottie } from 'vue3-lottie';
 import EmptySeed from '@/assets/images/lottie-empty-seed.json';
+import InputText from 'primevue/inputtext';
+import { FilterService } from 'primevue/api';
 
 const { isProjectConfigDialogVisible, menuProject } = useProjectMenu();
 
+const activeTabIndex = ref(0);
 const showVideo = ref(false);
+const searchProjects = ref<string>('');
```
Comment (YohannParis) [inline packages/client/hmi-client/src/page/Home.vue:188]:
```suggestion
const searchProjects = ref('');
```

File: packages/client/hmi-client/src/page/Home.vue
Comment (YohannParis) [inline packages/client/hmi-client/src/page/Home.vue:1]: All those sorting functions can be put in a `utils/project.ts`.

File: packages/client/hmi-client/src/page/Home.vue (hunk 2 above)
Comment (asylves1) [reply packages/client/hmi-client/src/page/Home.vue:62]: Looks like I found what happened to my old commits I was talking to you about in the morning. 
trying to clean this branch up now
